tutorials/openmm_tutorial.ipynb

```diff
@@ -0,0 +1,605 @@
+{
```

Comment:
This is awesome. But I think this should live in the docs folder somewhere. E.g., for jobflow we have the `jobflow/docs/tutorials` folder that contains python notebooks, and these are automatically built into the docs.

Reply:
Do you have a particular preference? I chose the location based on @jmmshn's tutorial PR but can definitely put it elsewhere. 

I would love to see automatically built jupyter notebooks for tutorials and would happily rewrite the docs into a notebook if it was available.